Applications/Spire/Source/Login/LoginWindow.cpp

```diff
@@ -33,6 +34,18 @@ namespace {
       set(Fill(QColor::fromRgb(0xE6, 0x3F, 0x45)));
     return style;
   }
+
+  auto INPUT_STYLE(StyleSheet style) {
```

Comment:
I think passing by reference is better here.

Reply:
The get_style function returns a const reference, so I can't pass it to INPUT_STYLE as a reference, and StyleSheet's get function returns a reference, so I would have to make a copy of style anyway before I could modify it if it was a const reference.

I'm not sure if const reference is always the standard, but I in this case I do like the implicit copy when passing by value.

Reply:
Make sense. I didn't see that the get_style function returns a const reference.